src/support/string.h

```diff
@@ -96,7 +96,10 @@ inline bool wildcardMatch(const std::string& pattern,
       return false;
```

Comment:
The comment on line 90-91 is now obsolete, please update it.

Reply:
Done.